src/index.ts

```diff
@@ -209,7 +209,7 @@ export class Squid {
 
   public setConfig(config: Config) {
     this.axiosInstance = axios.create({
-      baseURL: config.baseUrl || baseUrl,
+      baseURL: config.baseUrl ? config.baseUrl : baseUrl,
```

Comment:
i thought his error was
![Screen Shot 2022-12-09 at 13 37 45](https://user-images.githubusercontent.com/8565879/206704323-94102678-6746-4291-ad12-174c87505984.png)
which is different from your fix i believe

Reply:
maybe he's not using the latest version of sdk

Reply:
it is failing because of the `||` ?